src/administration/integrations/gluescript.md

```diff
@@ -0,0 +1,251 @@
+          'value': message,
+        }],
+      }],
+    }),
+  }
+);
+
+if (!resp.ok) { throw 'Failed'; }
+```
+
+where:
+* `SLACK_PAYLOAD_URL` is the URL provided by Slack for your custom webhook
+
+You can save the file to `.platform/scripts/my_glue_script.js` for example.
+
+### 2. Commit the script
+
+Commit the script to Git:
+
+```bash
+git add .platform/scripts/my_glue_script.js
+git commit -m "Committing the GlueScript file."
+git push platform master
+```
+
+where:
+* `my_glue_script.js` is the name of your Javascript file
+
+> Pushing the script to master won't do anything. Yet. Just a handy place to keep your GlueScript file.
+
+### 3. Enable the integration
+
+Enable the GlueScript integration as follows (you need to have the Platform.sh CLI installed):
+
+```bash
+platform integration:add --type script --file .platform/scripts/my_glue_script.js
+```
+
+where:
+* `my_glue_script.js` is the name of your Javascript file
+
+> **note**
+> This is the actual activation of the integration, the script is posted to the API through the CLI. Note that changing the file in Git will not change the integration. To update the integration you will need to use the CLI again.
+>
+> The GlueScript integration can only be managed by the project admin.
+
+# GlueScript APIs
+
+These are the internal Platform.sh APIs that are accessible through GlueScript, API calls are executed with the permissions of the user that created the integration.
+
+## Project API
+
+The project API gives you everything you need to know about the project in the context of which the GlueScript is running. 
+
+For example: `project.environments.master` will give you access to the metadata of the master environment of the current project.
+
+The API starts at `api.project` and is based on a one resource == one object model.
+
+### Properties
+
+The properties of the resource can be access directly on the object, like:
+
+```javascript
+project.owner
+```
+
+### Sub-objects
+
+Sub-objects can be accessed directly on the object, like:
+
+```javascript
+project.settings
+```
+
+### Collections
+
+Resources that are collections support iteration:
+
+```javascript
+project.environments.forEach(function(env) {
+  console.log(env.name);
+});
+```
+
+They also support direct access to an element:
+
+```javascript
+project.environments["master"].name
+```
+
+### Actions
+
+Resource actions are accessible as methods on the object:
+
+```javascript
+project.environments["master"].branch({
+  title="Test",
+  name="test",
+})
+```
+
+## Activity API
+
+The activity API accessed through `activity` gives you all the metadata about the current activity that invoked the GlueScript. The schema is the same as the one for Webhooks, documented in the [Webhook section](https://docs.platform.sh/administration/integrations/webhooks.html) of our documentation.
+
+## Process API
+The process API gives you access to the environment in which the GlueScript is running, such as to environment variables.
```

Comment:
Are these the environment variables of the actual project/environment? Since we make the distinction that the scripts don't "live in Git," it seems confusing to say that the script is running in an environment. My above review comment is a reflection of that confusion; I interpreted "environment" as something like a sandboxed execution environment given that it's the "process" API.

However, I'm not sure how I'd clarify this point, because the name of the API and the description of the API don't sound like they refer to the same thing.